commons/ava_testsuite/byzantine_gecko_network_tests.go

```diff
@@ -0,0 +1,115 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_services"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	BYZANTINE_CONFIG_ID = 2
+	BYZANTINE_USERNAME = "byzantine_gecko"
+	BYZANTINE_PASSWORD = "byzant1n3!"
+)
+// ================ Byzantine Test - Spamming Unrequested Chit Messages ===================================
+type StakingNetworkUnrequestedChitSpammerTest struct{
+	unrequestedChitSpammerImageName *string
+	normalImageName *string
+}
+func (test StakingNetworkUnrequestedChitSpammerTest) Run(network interface{}, context testsuite.TestContext) {
+	castedNetwork := network.(ava_networks.TestGeckoNetwork)
+	for i := 0; i < 4; i++ {
+		byzHighLevelClient, err := addServiceIdAsValidator(castedNetwork, i, BYZANTINE_USERNAME, BYZANTINE_PASSWORD, SEED_AMOUNT, STAKE_AMOUNT)
+		if err != nil {
+			context.Fatal(stacktrace.Propagate(err, "Failed to add Byzantine service as a validator."))
+		}
+		byzClient := byzHighLevelClient.GetLowLevelClient()
+		currentStakers, err := byzClient.PChainApi().GetCurrentValidators(nil)
+		if err != nil {
+			context.Fatal(stacktrace.Propagate(err, "Could not get current stakers."))
+		}
+		logrus.Debugf("Number of current stakers: %d", len(currentStakers))
+	}
+	availabilityChecker, err := castedNetwork.AddService(NORMAL_NODE_CONFIG_ID, 4)
```

Comment:
Rather than having this magic number here, can we at least name it as a variable, and where it's used below?